src/main/java/org/jboss/pnc/cleaner/builds/FailedBuildsCleaner.java

```diff
@@ -68,13 +70,15 @@
     @ConfigProperty(name = "failedbuildscleaner.indy.requesttimeout")
     int indyRequestTimeout;
 
-    private static List<BuildCoordinationStatus> failedStatuses;
+    private static List<BuildStatus> failedStatuses;
 
     static {
-        failedStatuses = new ArrayList<>(2);
-        failedStatuses.add(BuildCoordinationStatus.CANCELLED); // added for old data, now cancellation cleans up the Indy data
-        failedStatuses.add(BuildCoordinationStatus.DONE_WITH_ERRORS);
-        failedStatuses.add(BuildCoordinationStatus.SYSTEM_ERROR);
+        failedStatuses = new ArrayList<>(5);
+        failedStatuses.add(BuildStatus.CANCELLED); // added for old data, now cancellation cleans up the Indy data
+        failedStatuses.add(BuildStatus.SYSTEM_ERROR);
+        failedStatuses.add(BuildStatus.FAILED);
+        failedStatuses.add(BuildStatus.REJECTED_FAILED_DEPENDENCIES);
```

Comment:
Hmm, why were these added? The rejected builds were not started, so they did not create any repos in Indy. It doesn't make sense to have them here.

Reply:
Good point. You are right. Removing them.